### PR TITLE
Kogito 2549 - fix metrics example

### PR DESCRIPTION
**Pull request by r00ta**
Jira link: https://issues.redhat.com/browse/KOGITO-2549

This PR fixes the readme for the kogito metrics example, optimize code/imports and fix a licence header.

Many thanks for submitting your Pull Request :heart:! 

Please make sure that your PR meets the following requirements:

**WARNING! Please make sure you are opening your PR against `master` branch!**

- [x] You have read the [contributors guide](https://github.com/kiegroup/kogito-runtimes#contributing-to-kogito)
- [x] Pull Request title is properly formatted: `KOGITO-XYZ Subject`
- [] Pull Request title contains the target branch if not targeting master: `[0.9.x] KOGITO-XYZ Subject`
- [x] Pull Request contains link to the JIRA issue
- [x] Pull Request contains link to any dependent or related Pull Request
- [x] Pull Request contains description of the issue
- [x] Pull Request does not include fixes for issues other than the main ticket

### CODE REVIEW

File: dmn-drools-quarkus-metrics/src/main/java/org/kie/kogito/examples/Hello.java [hunk 1]
```diff
@@ -20,6 +20,7 @@
 import org.kie.kogito.rules.RuleUnitData;
 
 public class Hello implements RuleUnitData {
+
```
Comment (danielezonca) [inline dmn-drools-quarkus-metrics/src/main/java/org/kie/kogito/examples/Hello.java:23]: Thanks for your contribution! Well done! :smile:

File: dmn-drools-quarkus-metrics/src/main/java/org/kie/kogito/examples/MyInterceptor.java [hunk 2]
```diff
@@ -12,8 +12,7 @@
  *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
  *  See the License for the specific language governing permissions and
  *  limitations under the License.
- */
-package org.kie.kogito.examples;
+ */package org.kie.kogito.examples;
```
Comment (danielezonca) [inline dmn-drools-quarkus-metrics/src/main/java/org/kie/kogito/examples/MyInterceptor.java]: New line?

File: dmn-drools-quarkus-metrics/src/main/java/org/kie/kogito/examples/Hello.java (hunk 1 above)
Comment (r00ta) [reply dmn-drools-quarkus-metrics/src/main/java/org/kie/kogito/examples/Hello.java:23]: Thanks Intellij's Ctrl+Alt + O  :)

**Review by danielezonca — APPROVED** [review]

**Review by jiripetrlik — APPROVED** [review]
Thank you! Looks good to me!

**Comment by danielezonca** [comment]
jenkins retest this please

**Pull request merged**